### DISCUSSION TITLE
Issue - Unexpected Mail - SSL Error

### DISCUSSION THREAD

**Discussion by nicogithub22** [globaleaks/globaleaks-whistleblowing-software#3544] [Q&A] [answered]
### What version of GlobaLeaks are you using?

4.11.5

### What browser(s) are you seeing the problem on?

All

### What operating system(s) are you seeing the problem on?

Windows

### Describe the issue

I get this Mail:

Version: 4.11.5

twisted.web._newclient.ResponseNeverReceived 
    A L{ResponseFailed} that knows no response bytes at all have been received.
    

Traceback (most recent call last):

  File "/usr/lib/python3/dist-packages/globaleaks/jobs/job.py", line 49, in run
    yield self.operation()

twisted.web._newclient.ResponseNeverReceived: [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', '', 'unexpected eof while reading')]>]

What does this error message mean, what can I do about it?

### Proposed solution

Check SSL Certificate, thats correct.

**Comment by nicogithub22** [comment]
Can I get a short answer, please.

**Comment by evilaliv3** [comment, marked as answer]
Hello!

How many times a day do you get this error?

If it is once I day it could be related to our repository that recently had
an expired SSL certificate
